tests/hil/tests/lightdb/test.c

```diff
@@ -395,7 +395,11 @@ static void test_lightdb_invalid(struct golioth_client *client)
 
 static void test_lightdb_observe_setup(struct golioth_client *client)
 {
-    golioth_lightdb_observe_async(client, "hil/lightdb/observed/int", int_cb, NULL);
+    golioth_lightdb_observe_async(client,
```

Comment:
Let's add a test for CBOR observations too

Reply:
I've added a commit to test both CBOR and JSON observation types.